src/models/template.ts

```diff
@@ -9,7 +9,6 @@ const TemplateFormRuntype = Rt.Record({
 
 const TemplateManifestRuntype = Rt.Record({
   filePath: Rt.String,
-  fileRenameRule: Rt.String,
```

Comment:
wouldn't this be good to have here as well?

Reply:
We can reintroduce it later if we allow saving the template output directly to files.
But for now, we're just creating unsaved resources from templates so this field is not needed